### PR TITLE
JavaCV + Android live streaming example

### PR DESCRIPTION
**Pull request by mmcc**

### CODE REVIEW

File: android-live-javacv/.gitignore [hunk 1]
```diff
@@ -0,0 +1,9 @@
+*.iml
+.gradle
+/local.properties
+/.idea/workspace.xml
+/.idea/libraries
+.DS_Store
+/build
+/captures
+.externalNativeBuild
```
Comment (jsanford8) [inline android-live-javacv/.gitignore]: I think you may want to gitignore everything in `.idea`? This is what we do in ExoPlayer, and it seems like it's good: https://github.com/muxinc/mux-stats-sdk-exoplayer/blob/master/.gitignore

File: android-live-javacv/app/src/main/java/com/example/mmcc/javacvexample/MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,531 @@
...
+import org.bytedeco.javacv.Frame;
+import org.bytedeco.javacv.FrameFilter;
+
+public class MainActivity extends Activity implements OnClickListener {
+
+    private final static String CLASS_LABEL = "MainActivity";
+    private final static String LOG_TAG = CLASS_LABEL;
+
+    /* This isn't a live RTMP endpoint. You should replace this line with your own! */
+    private String ffmpeg_link = "rtmp://live.mux.com/app/f5212894-e8ee-9eba-02e9-ac21c6249e0b";
```
Comment (jsanford8) [inline android-live-javacv/app/src/main/java/com/example/mmcc/javacvexample/MainActivity.java]: you deleted this live stream, right?

Comment (jsanford8) [reply android-live-javacv/app/src/main/java/com/example/mmcc/javacvexample/MainActivity.java]: May want to put in like a `YOUR_STREAM_KEY`

File: android-live-javacv/app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="app_name">JavaCVExample</string>
```
Comment (jsanford8) [inline android-live-javacv/app/src/main/res/values/strings.xml]: Should we take a chance and rename this?

File: android-live-javacv/app/src/main/java/com/example/mmcc/javacvexample/MainActivity.java (hunk 2 above)
Comment (mmcc) [reply android-live-javacv/app/src/main/java/com/example/mmcc/javacvexample/MainActivity.java]: Yeah that stream is deleted, but I updated the name.

**Pull request merged**